main.nf

```diff
@@ -486,27 +581,18 @@ if( params.skip_trimming ){
         file "where_are_my_files.txt"
 
         script:
-        def c_r1 = clip_r1 > 0 ? "--clip_r1 $clip_r1" : ''
```

Comment:
why did you remove the multicore support?

Reply:
I think I just forgot to copy the newer version of methylseq to this process. Fixed